Python/bytecodes.c

```diff
@@ -813,6 +813,13 @@ dummy_func(
         };
 
         inst(SEND, (unused/1, receiver, v -- receiver, retval)) {
+            if ((frame->owner == FRAME_OWNED_BY_GENERATOR) &&
+                (frame->f_code->co_flags & (CO_COROUTINE | CO_ASYNC_GENERATOR))) {
```

Comment:
Why do we use different conditions here than we use in `SEND_GEN` below, where we check only the type of `receiver`?

Reply:
Check should be the same, good catch!